code/mob/living/carbon/human/procs/emote.dm

```diff
@@ -2277,6 +2297,7 @@
 // that would be difficult, datumize emotes 2day!
 #ifdef SPACEMAN_DMM
 /mob/living/carbon/human/emote(var/act, var/voluntary = 0, var/emoteTarget = null)
+	..()
```

Comment:
I think this line might be what's screwing up the linter

Reply:
oh no it's worse than that